src/appliance-onboarding-script/appliance_setup/avs/avsarconboarder/orchestrator/network/_network_orchestrator.py

```diff
@@ -0,0 +1,49 @@
+from ...orchestrator._orchestrator import Orchestrator
+import ipaddress
+from avs.avsarconboarder.exception import InvalidInputError
+
+class NetworkOrchestrator(Orchestrator):
+    def __init__(self):
+        self.list_of_ip_addr_in_segment = list()
+
+    def validate_static_ip_cidr_block(self, config):
+        segmant_ip_cidr = config["staticIpNetworkDetails"]["networkCIDRForApplianceVM"]
```

Comment:
>segmant_ip_cidr [](http://example.com/codeflow?start=8&length=15)

if the segment ip cidr is different from the ip cidr of the segment provided, what will be the behavior? Do we need to add some validation?

Reply:
Are we going to validate if the IP address block and name provided in config file and any segment (with same name or ip range is already present as nsx-t segment) are matching? Today we do not have such validation. shall we take up as a different task?

Reply:
added scenarios for various validations in config

Reply:
We validated if the segment name and ip cidr value are same in the config file and nsx-t segment is matching